grafana/k6-studio
# feat: word-wrap editor

## Description

This PR enables word-wrap for the Monaco editor in Script Preview and the Content tab of a response. The state for both are persisted individually in the settings file.



## How to Test

- open Script Preview and toggle the word-wrap switch
- open the response content of a JSON request and toggle word-wrap
- re-open the app and check that the previous state was persisted



## Checklist

- [] I have added tests for my changes.


## Screenshots (if appropriate):

https://github.com/user-attachments/assets/3b93ffa2-ed78-458d-9ae9-108ab4c01f58


## Related PR(s)/Issue(s)





Resolves https://github.com/grafana/k6-studio/issues/237
